### PR TITLE
feat: exercise05

### PR DESCRIPTION
**Pull request by Bateyjosue**
# Task

- [ ] Convert to binary
- [ ] calculate fibonacci

## Cheer 🎉 🎉

### CODE REVIEW

File: src/Exercise05/calculate-fibonacci.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const calculateFibonacci = (number) => {
+    if (number <= 0) return [];
+    if (number === 1) return [0];
+    if (number === 2) return [0, 1];
+
+    const fibonaccis = calculateFibonacci(number - 1);
```
Comment (Nkaka23dev) [inline src/Exercise05/calculate-fibonacci.js]: why call it fibonaccis? it is better if you find a good name

File: src/Exercise05/convert-binary.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const convertBinary = (number) => {
+    return number.toSttring(2)
+}
```
Comment (Nkaka23dev) [inline src/Exercise05/convert-binary.js]: you don't need return keyword and curly braces

**Review by Nkaka23dev — CHANGES_REQUESTED** [review]

File: src/Exercise05/calculate-fibonacci.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const calculateFibonacci = (number) => {
+    if (number <= 0) return [];
+    if (number === 1) return [0];
+    if (number === 2) return [0, 1];
+
+    const fibonacciSeries = calculateFibonacci(number - 1);
+    fibonacciSeries.push(fibonacciSeries.at(-1) + fibonacciSeries.at(-2));
+    return fibonacciSeries;
```
Comment (Bateyjosue) [inline src/Exercise05/calculate-fibonacci.js:8]: Add comments

File: src/Exercise05/convert-binary.js [hunk 4]
```diff
@@ -0,0 +1 @@
+const convertBinary = (number) => number.toString(2)
```
Comment (Bateyjosue) [inline src/Exercise05/convert-binary.js]: remove paranthesis

**Pull request merged**